app/charts/shared/axis-height-linear.tsx

```diff
@@ -61,7 +64,10 @@ export const AxisHeightLinear = () => {
           {state.yAxisLabel}
         </text>
       ) : (
-        <foreignObject width={axisTitleWidth} height={axisLabelFontSize * 2}>
+        <foreignObject
+          width={textIsOverlapping ? state.bounds.chartWidth : axisTitleWidth}
+          height={axisLabelFontSize * overlappingAmount * 2}
+        >
```

Comment:
@noahonyejese thanks for improving this 💯

I think there's still a need to increase the chart height / margin between the axis and the graphical elements – now it's already better, as it wraps, but we can end up with a situation like this:

<img width="277" alt="Screenshot 2024-09-16 at 08 58 56" src="https://github.com/user-attachments/assets/15c4b91c-03af-469d-969b-1e7e35b04321">

Reply:
@noahonyejese I am not sure if this fixed the problem, I think you need to also update chart bounds to take into account the extended chart height, like you did for [combo charts](https://github.com/visualize-admin/visualization-tool/blob/f18afecffe67893e79010db1779f223deaca82ac/app/charts/combo/combo-line-dual-state.tsx#L215-L217)?

<img width="276" alt="Screenshot 2024-09-22 at 13 04 37" src="https://github.com/user-attachments/assets/5e7f552e-2b22-4767-9e88-cc09f6c61563">